src/hb-glib.cc

```diff
@@ -371,6 +397,16 @@ void free_static_glib_funcs ()
 }
 #endif
 
+/**
+ * hb_glib_get_unicode_funcs:
+ *
+ * Fetches a Unicode-functions structure that is populated
+ * with the appropriate GLib function for each method.
+ *
+ * Return value: (transfer full): a pointer to the #hb_unicode_funcs_t Unicode-functions structure
```

Comment:
```suggestion
 * Return value: (transfer none): a pointer to the #hb_unicode_funcs_t Unicode-functions structure
```
The `_get_` functions return a reference unlike the `_copy_` functions, so the user must not destroy the returned object.